qiskit_metal/_gui/widgets/create_component_window/parameter_entry_window.py

```diff
@@ -165,6 +165,14 @@ def _setup_help(self):
             raise Exception("No Component found.")
 
         filepath = self.qcomponent_file_path
+
+        imageStringPath = filepath.replace(".py", ".png")
+        imagefilename = os.path.basename(os.path.normpath(imageStringPath))
+        for i in range(3):
```

Comment:
Why is this in range 3?

Reply:
are you trying to go up a level; just use pathlib for that : ` path.parent` see https://realpython.com/python-pathlib/